extensions/wikia/PortableInfoboxBuilder/PortableInfoboxBuilderHooks.class.php

```diff
@@ -65,7 +65,8 @@ public static function onMakeGlobalVariablesScript( array &$aVars ) {
 
 		if ( $wgEnableVisualEditorExt && \VisualEditorHooks::isAvailable( \RequestContext::getMain()->getSkin() ) ) {
 			$aVars['wgEnablePortableInfoboxBuilderInVE'] = $wgEnablePortableInfoboxBuilderInVE &&
-				\RequestContext::getMain()->getUser()->isAllowed( 'createpage' );
+				// use dummy title in template ns
```

Comment:
Write it in the code ;) `// use dummy title in template ns` isn't helpful, it should say _why_ we do this, and not duplicate what is already visible.